From 5d3e611d962a0d976db6bc7a5f2c7e2a9cb5f5cc Mon Sep 17 00:00:00 2001
From: Pardha Kesapragada <pkesapragada@habana.ai>
Date: Fri, 12 Feb 2021 18:22:16 +0200
Subject: [PATCH] [SW-24746] gaudi2: set interrupt mask to default

This change -
"[SW-24746] preboot: set interrupt mask to default"
was not done for Gaudi2. Hence it caused a compile
error. Porting the same change to fix the error.

Change-Id: I1391a2434ae4d20d547214f005b428d8db4461d7
Signed-off-by: Pardha Kesapragada <pkesapragada@habana.ai>
---

diff --git a/gaudi2/common.h b/gaudi2/common.h
index ce05871..f1a2afd 100644
--- a/gaudi2/common.h
+++ b/gaudi2/common.h
@@ -89,5 +89,7 @@
 #define PCIE_PHY_PROTOCOL0_EXT_RX_ADAPT_MODE_G3_CONF	0x5555
 #define PCIE_PHY_PROTOCOL0_EXT_RX_ADAPT_SEL_G4_CONF	0xFF
 
+#define I2C_SLAVE_DEFAULT_INTR_MASK			0x48FF
+
 #endif /* COMMON_GAUDI2_H_ */
 
